pkg/azurefile/controllerserver.go

```diff
@@ -88,18 +89,19 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		fileShareName = getValidFileShareName(name)
 	}
 
-	if exists, err := d.checkFileShareExists(account, resourceGroup, fileShareName); err != nil {
+	if exists, err := d.checkFileShareExists(account, resourceGroup, fileShareName, accountKey); err != nil {
 		return nil, err
 	} else if exists {
 		volumeID = fmt.Sprintf(volumeIDTemplate, resourceGroup, account, fileShareName)
```

Comment:
I think the `checkFileShareCapacity` process only happens when file share exists, so what about merge `checkFileShareCapacity` and `checkFileShareExists` into one function?